### ISSUE TITLE
Navigation List View: Drag and drop considerations

### ISSUE THREAD

**Issue by scruffian** [WordPress/gutenberg#45448]
As outlined in https://github.com/WordPress/gutenberg/issues/42257, the Navigation List View should allow users to reorder Navigation Items using drag and drop

This is made complex because when Navigation Items are moved around, a new "Submenu" block needs to be created to hold the child items.

There is an accessibility consideration here too, since keyboard users cannot use drag and drop.

**Comment by annezazu** [comment]
Noting some light usability feedback with the current experience from the [nineteenth call for testing for the FSE Outreach Program:](https://make.wordpress.org/test/2022/12/14/fse-program-testing-call-19-running-through-refinements/#comment-2812)

>Dragging a link to create a submenu should be easier. Currently one needs to have the cursor on the blue line and move it to the right to create a submenu. Current approach reminds me too much of the nudging in the classic menu screen.

[& more here:](https://make.wordpress.org/test/2022/12/14/fse-program-testing-call-19-running-through-refinements/#comment-2828)

>I found the dragging dropping of menu items in the right sidebar a bit eventful. Had a hard time positioning where I wanted them. If I accidentally indented them an extra level, I couldn’t go back just one level – I had to move it to the main menu, then reattempt to add it as a 1st level submenu. That said, it’s waaaaaay more clear and manageable than before so I think this is a significant improvement.

**Comment by scruffian** [comment]
Drag and drop is mostly working well, we can close this.
